plugins/woocommerce/src/Internal/Admin/SystemStatusReport.php

```diff
@@ -0,0 +1,215 @@
+				<thead>
+				<tr>
+					<th colspan="5" data-export-label="Admin">
+						<h2>
+							<?php esc_html_e( 'Admin', 'woocommerce' ); ?><?php echo wc_help_tip( esc_html__( 'This section shows details of WC Admin.', 'woocommerce' ) ); ?>
+						</h2>
+					</th>
+				</tr>
+				</thead>
+				<tbody>
+					<?php
+						$this->render_features();
+						$this->render_daily_cron();
+						$this->render_options();
+						$this->render_notes();
+						$this->render_onboarding_state();
+					?>
+				</tbody>
+			</table>
+		<?php
+	}
+
+	/**
+	 * Render features rows.
+	 */
+	public function render_features() {
+		/**
+		 * Filter the admin feature configs.
+		 *
+		 * @since 6.5.0
+		 */
+		$features          = apply_filters( 'woocommerce_admin_get_feature_config', wc_admin_get_feature_config() );
+		$enabled_features  = array_filter( $features );
+		$disabled_features = array_filter(
+			$features,
+			function( $feature ) {
+				return empty( $feature );
+			}
+		);
+
+		?>
+			<tr>
+				<td data-export-label="Enabled Features">
+					<?php esc_html_e( 'Enabled Features', 'woocommerce' ); ?>:
+				</td>
+				<td class="help"><?php echo wc_help_tip( esc_html__( 'Which features are enabled?', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
+				<td>
+					<?php
+						echo esc_html( implode( ', ', array_keys( $enabled_features ) ) )
+					?>
+				</td>
+			</tr>
+
+			<tr>
+				<td data-export-label="Disabled Features">
+					<?php esc_html_e( 'Disabled Features', 'woocommerce' ); ?>:
+				</td>
+				<td class="help"><?php echo wc_help_tip( esc_html__( 'Which features are disabled?', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
+				<td>
+					<?php
+						echo esc_html( implode( ', ', array_keys( $disabled_features ) ) )
+					?>
+				</td>
+			</tr>
+		<?php
+	}
+
+
+	/**
+	 * Render daily cron row.
+	 */
+	public function render_daily_cron() {
+		$next_daily_cron = wp_next_scheduled( 'wc_admin_daily' );
+		?>
+			<tr>
+				<td data-export-label="Daily Cron">
+					<?php esc_html_e( 'Daily Cron', 'woocommerce' ); ?>:
+				</td>
+				<td class="help"><?php echo wc_help_tip( esc_html__( 'Is the daily cron job active, when does it next run?', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
+				<td>
+					<?php
+					if ( empty( $next_daily_cron ) ) {
+						echo '<mark class="error"><span class="dashicons dashicons-warning"></span> ' . esc_html__( 'Not scheduled', 'woocommerce' ) . '</mark>';
+					} else {
+						echo '<mark class="yes"><span class="dashicons dashicons-yes"></span> Next scheduled: ' . esc_html( date_i18n( 'Y-m-d H:i:s P', $next_daily_cron ) ) . '</mark>';
+					}
+					?>
+				</td>
+			</tr>
+		<?php
+	}
+
+	/**
+	 * Render option row.
+	 */
+	public function render_options() {
+		$all_options_expected = is_numeric( get_option( 'woocommerce_admin_install_timestamp' ) )
```

Comment:
Small thing but I think it's worth making sure that `woocommerce_admin_install_timestamp` is more than 0

Reply:
Good point! Updated in 902dcf3369212732fcb9490260660b459e888f08